src/templates/components/getUpdatesSection/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { slugifyString } from '@/lib/utils/slug'
+
+type GetUpdatesSectionProps = {
+  links: {
+    label: string
+    url: string
+    type: string | null
+  }[]
+  heading: string
```

Comment:
Heading could be a bit more generic like `ReactNode`, but it doesn't need to change until we actually need it changed